src/lib/rendererHelper.ts

```diff
@@ -35,6 +35,20 @@ export async function renderImage(actionContext: string, state: ActionState) {
 		image.onload = resolve;
 	});
 
+	let alertImage = document.createElement("img");
```

Comment:
These images should only be loaded inside their respective if statements below to prevent unnecessary network usage and performance impact. Additionally, please use a consistent order; `ok` seems to come before `alert` in the existing codebase, the same way you've added them to the signature of this function.